.all-contributorsrc

```diff
@@ -152,6 +152,16 @@
       "contributions": [
         "doc"
       ]
+    },
+    {
```

Comment:
@GenerousDev I see you included your details in the .all-contributorsrc file. This should not be so. This file is auto-generated once your PR is approved by an admin.

Could you remove the changes made to this file so we can review your PR?